requirements-pulp.yml

```diff
@@ -1,2 +1,4 @@
 collections:
-  - pulp.pulp_installer
+  - name: https://github.com/ehelms/foreman-operations-collection
```

Comment:
this should go to `requirements.yml` or something I guess?

Reply:
Dang me. Reading is hard on fridays.